src/backend/aspen/api/config/config.py

```diff
@@ -70,9 +70,21 @@ def SECRET_KEY(self) -> str:
     def AUTH0_CLIENT_ID(self) -> str:
         return self.AWS_SECRET["AUTH0_CLIENT_ID"]
 
+    @property
+    def AUTH0_LOGOUT_URL(self) -> str:
+        flask_env = os.environ.get("FLASK_ENV")
+        if flask_env == "production":
+            return f"{self.AUTH0_BASE_URL}/v2/logout"
+        else:
+            return f"https://{self.AUTH0_DOMAIN}/Account/Logout"
+
     @property
     def AUTH0_CALLBACK_URL(self) -> str:
-        raise NotImplementedError()
+        flask_env = os.environ.get("FLASK_ENV")
+        if flask_env == "production":
+            return f"{os.getenv('API_URL', '')}/v2/auth/callback"
```

Comment:
why do we have the default as an empty string? will this error if that env variable is not present and no default was provided?

Reply:
if so i would rather we have it error, it seems more clear that way?

Reply:
It's a bad hack to make mypy happy.... I'll do something smarter. 😅

Reply:
mypy 😠